lib/rust/ensogl/component/scrollbar/src/lib.rs

```diff
@@ -53,6 +56,10 @@ pub const PADDING: f32 = 3.0;
 const MIN_THUMB_SIZE: f32 = 12.0;
 /// After an animation, the thumb will be visible for this time, before it hides again.
 const HIDE_DELAY: f32 = 1000.0;
+/// Time delay before holding the mouse button triggers scrolling.
+const HOLD_REPEAT_DELAY: Duration = Duration::from_millis(500);
+/// Time interval between scrolls when holding the mouse button.
+const HOLD_REPEAT_INTERVAL: Duration = Duration::from_millis(200);
```

Comment:
I'm having trouble using the `Duration` type from unit library in const contexts, because it is using traits to define all of its methods (const traits are unstable and seem to break on use here).

Also, that type stores the time as a `f32`, which introduces conversion to i32 on each use. Also it gives a false idea of timer accuracy. This is especially important for `Interval` timer, as the rounding error will accumulate over the timer runtime.

For that reason, I'm going to use `i32` directly here, as that's what web-sys expects from me. The alternative would be to define new integer time duration type, but I don't think it would find much use across the codebase. The browser timers are pretty special in that regard.

Reply:
I'm totally ok with using i32 here for now.